tailwind.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
         "table-data": "#ebecf2",
         "hover-bg": "#222329",
         "tooltip-border": "#252833",
-        subheading: "#828699",
```

Comment:
I feel like we keep changing this back and forth hah. The analytics designs must have a different "dimmed/helper" text